stdlib/database/h2/src/test/java/org/ballerinalang/stdlib/database/sql/SQLActionsTest.java

```diff
@@ -359,18 +359,27 @@ public void testComplexTypeRetrieval() {
     public void testFailedSelect() {
         BValue[] returns = BRunUtil.invoke(resultNegative, "testSelectData");
         Assert.assertTrue(returns[0].stringValue().contains("execute query failed:"));
+        Assert.assertTrue(returns[0].stringValue().contains("{ballerina\\/sql}DatabaseError"));
+        Assert.assertTrue(returns[0].stringValue().contains("sqlErrorCode:"));
+        Assert.assertTrue(returns[0].stringValue().contains("sqlState:"));
```

Comment:
Shall we keep these tests as they are, and add a few new tests (IMO it is OK to duplicate these tests if needed) and verify that Database/Application errors are thrown whenever applicable?